support/globalconfig/ingress.go

```diff
@@ -18,12 +18,17 @@ func IngressConfig() *configv1.Ingress {
 }
 
 func ReconcileIngressConfig(cfg *configv1.Ingress, hcp *hyperv1.HostedControlPlane, globalConfig GlobalConfig) {
-	cfg.Spec.Domain = IngressDomain(hcp)
+	cfg.Spec.Domain = IngressDomain(hcp, globalConfig)
 	if globalConfig.Ingress != nil {
 		cfg.Spec = globalConfig.Ingress.Spec
 	}
 }
 
-func IngressDomain(hcp *hyperv1.HostedControlPlane) string {
-	return fmt.Sprintf("apps.%s", BaseDomain(hcp))
+func IngressDomain(hcp *hyperv1.HostedControlPlane, globalConfig GlobalConfig) string {
```

Comment:
Instead of passing GlobalConfig, we just need *configv1.Ingress

Reply:
done